tools/ot/src/test/java/dk/alexandra/fresco/tools/ot/base/TestFunctionalNaorPinkas.java

```diff
@@ -49,10 +70,14 @@ public void shutdown() {
 
   private List<Pair<StrictBitVector, StrictBitVector>> otSend(int iterations) throws Exception {
     Network network =
-        new CheatingNetworkDecorator(new SocketNetwork(RuntimeForTests.defaultNetworkConfiguration(1, Arrays.asList(1, 2))));
+        new CheatingNetworkDecorator(
+            new SocketNetwork(RuntimeForTests.defaultNetworkConfiguration(1, Arrays.asList(1, 2))));
     try {
       Drbg rand = new AesCtrDrbg(HelperForTests.seedOne);
-      Ot otSender = new NaorPinkasOt(2, rand, network, staticParams);
+      Class clazz = this.testClass;
+      Constructor[] constructors = clazz.getConstructors();
+      Ot otSender = (AbstractNaorPinkasOT) constructors[0]
+          .newInstance(2, rand, network);
```

Comment:
Used to get the constructor via reflection. Is not the most beautiful solution, but the best I could come up with without changing too much of the existing code